src/figuresClasses.ts

```diff
@@ -1,19 +1,73 @@
-export interface Figure {
 
+export interface Figure {
+  shape: 'triangle' | 'circle' | 'rectangle';
+  color: 'red' | 'green' | 'blue';
```

Comment:
```suggestion
  shape: 'triangle' | 'circle' | 'rectangle';
  color: 'red' | 'green' | 'blue';
```
Use enums for shape and color